main/webapp/modules/core/scripts/views/data-table/column-header-ui.js

```diff
@@ -69,7 +69,35 @@ DataTableColumnHeaderUI.prototype._render = function() {
   elmts.dropdownMenu.on('click',function() {
     self._createMenuForColumnHeader(this);
   });
-  
+
+  var serviceUrl = null;
+  var service = null;
+  var serviceLogo=null;
+  if (this._column.reconConfig) {
+   serviceUrl =this._column.reconConfig.service;
+  }
+  try {
+    
+    if (new URL(serviceUrl)) {
+      service = ReconciliationManager.getServiceFromUrl(serviceUrl);
+    }
+    if(service) {
+      serviceLogo=service.logo;
+   }
+
+    var img =$("<img>");
+    if(serviceLogo) {
+      var imageUrl = serviceLogo;
+      img.attr("src", imageUrl);
+      img.attr("title", service.name);
+      img.addClass("serviceLogo")
+      img.appendTo(elmts.serviceLogoContainer.show());
+   }
+  }
+  catch {
+    console.log("The URL is not valid");
```

Comment:
Those logging statements could be more useful with a more explicit error message. When you are looking at the code here, it is clear that this logging statement refers to a recon service URL, but imagine what happens when just seeing this message in the console, in the middle of other unrelated statements: the current error message does not give a lot of information, as we have no idea which URL is referred to.

Once the `try { } catch { }` block is part of the new function suggested above, this logging statement will be about the validity of the logo URL, hence we could instead have something like `console.warn("Invalid logo URL supplied by service "+serviceUrl)` or something like that.

Reply:
It seems like this problem is still there, no? Or do you see it resolved in another way?